.github/workflows/auto_cherry_pick.yml

```
@@ -7,37 +7,18 @@ on:
       - closed
 
 jobs:
-  branch-matrix:
-    if: github.event.pull_request.merged == true && contains(github.event.pull_request.labels.*.name, 'CherryPick')
-    name: Generate a branch matrix to apply cherrypicks
-    runs-on: ubuntu-latest
-    outputs:
-      branches: ${{ steps.set-matrix.outputs.branches }}
-    steps:
-      - uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-      - id: set-matrix
-        run: echo "::set-output name=branches::$(git branch -rl --sort=-authordate 'origin/6.*.z' --format='%(refname:lstrip=-1)' | head -n2 | jq -cnR '[inputs | select(length>0)]')"
   auto_cherry_picking:
     if: github.event.pull_request.merged == true && contains(github.event.pull_request.labels.*.name, 'CherryPick')
-    name: Auto Cherry picking to branches
-    needs: branch-matrix
+    name: Auto Cherry picking to 6.11.z branch
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        to_branch: ${{ fromJson(needs.branch-matrix.outputs.branches) }}
     steps:
       - name: Checkout Robottelo
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
-        if: matrix.to_branch != github.base_ref
-      - name: Cherry pick into ${{ matrix.to_branch }}
-        uses: jyejare/github-cherry-pick-action@main
+      - name: Cherry pick into 6.11.z
+        uses: jyejare/github-cherry-pick-action@1.0.4
```

Any reason to hardcode this version here? It would be better to always use the `main` to pick up the latest code
```suggestion
        uses: jyejare/github-cherry-pick-action@main
```

---

We dont want to be on latest because we are not testing everything we are merging and hence tagging works best with the tested code.